__tests__/app.test.js

```diff
@@ -30,21 +29,73 @@ describe('* invalid endpoints', () => {
         .get('/doesn"t/exist')
         .expect(404)
         .then((res) => {
-            console.log(res.body.msg)
             expect(res.body.msg).toEqual('Endpoint not found')
-
+            
         })
     })
-
+    
     it('returns a 404 status code when making an invalid put request', () => {
         return request(app)
         .get('/doesn"t/exist')
         .send({msg: 'This should message should never '})
-        .expect(404)
         .then((res) => {
-            console.log(res.body.msg)
             expect(res.body.msg).toEqual('Endpoint not found')
+            
+        })
+    })
+})
+
+describe('GET /api', () => {
+    it('returns a 200 status code and returns an object of available end points', () => {
+        return request(app)
+        .get('/api')
+        .expect(200)
+        .then((res) => {
+            const endPoints = res.body.newEndPoints
+            expect(res.headers['content-type']).toMatch(/application\/json/)
+            expect(endPoints).toHaveProperty('GET /api')
+            expect(endPoints).toHaveProperty('GET /api/topics')
+            expect(endPoints).toHaveProperty('GET /api/articles')
 
         })
     })
 })
+
+describe.only('GET /api/articles/:article_id', () => {
+    it('returns a 200 status code when valid id given and returns an object with correct properties', () => {
+        return request(app)
+        .get('/api/articles/1')
+        .expect(200)
+        .then((res) => {
+            const article = res.body.article[0]
+            expect(Object.keys(article)).toHaveLength(8)
+            
+            expect(article).toHaveProperty('author', expect.any(String));
+            expect(article).toHaveProperty('title', expect.any(String));
+            expect(article).toHaveProperty('article_id', expect.any(Number));
+            expect(article).toHaveProperty('body', expect.any(String));
+            expect(article).toHaveProperty('topic', expect.any(String));
+            expect(article).toHaveProperty('created_at', expect.any(String)); 
+            expect(article).toHaveProperty('votes', expect.any(Number));
+            expect(article).toHaveProperty('article_img_url', expect.any(String)); 
```

Comment:
It's Ok to check the shape of the object, but as we know which article will come back, it'll be better if the assertion could be more specific. To check the article is the exact article we're expecting